packages/media/src/webrtc/P2pRtcManager.ts

```diff
@@ -224,10 +225,6 @@ export default class P2pRtcManager implements RtcManager {
         return this._replaceTrackToPeerConnections(oldTrack, newTrack);
     }
 
-    accept({ clientId, shouldAddLocalVideo }: { clientId: string; shouldAddLocalVideo?: boolean }) {
```

Comment:
Was this never called then?

Reply:
I did some archeology to try to find out. Maybe way back when SFU and P2P shared the same BaseRtcManager in a meaningful way? If we're using it, there should be a test failing somewhere using it, other than the tests that just verify the implementation.

Reply:
Gotcha 👍